cmd/podman/main.go

```diff
@@ -26,7 +26,10 @@ func main() {
 	if reexec.Init() {
 		return
 	}
-
+	cli.HelpFlag = cli.BoolFlag{
```

Comment:
do you need this?  I think it's provided by default by urfavce/cli.

Reply:
I don't think you need this.